### DISCUSSION TITLE
how to bookmark a server?

### DISCUSSION THREAD

**Discussion by luckman212** [shlinkio/discussions#22] [Help wanted] [answered]
### Shlink version

3.6.4

### PHP version

-

### How do you serve Shlink

RoadRunner Docker image

### Database engine

MariaDB

### Database version

latest

### Summary

I have 2 computers, and set up a selfhosted Shlink instance in Docker. I _think_ it's all working.

My question is, how can I "bookmark" a path to my frontend? On each machine, the UUID is different, so if I bookmark it on one, the link does not open on the other computer.

The UUID I am referring to is e.g.

```
https://shlink.my.hostname/server/c6377f6b-7a2b-4d08-9c40-5ba2f406c401/overview
                 uuid ____________^
```

**Comment by acelaya** [comment, marked as answer]
> My question is, how can I "bookmark" a path to my frontend?

I'm afraid you can't.

shlink-web-client runs on the browser, so every instance is different.

Server IDs are randomly generated and not shared, so you can't get a predictable URL for one specific server.

This [was discussed](https://github.com/shlinkio/shlink-web-client/issues/435) some time ago, and there was even [an attempt](https://github.com/shlinkio/shlink-web-client/pull/437) at addressing it, but I had some concerns with that approach.

That PR includes my suggestion at how this could be reasonable done, if you want to give it a try.

Additionally, there will eventually be a new Shlink client (so called Shlink Dashboard), which will have it's own backend, and servers will be shared among all consumers. However, this will still take some time.

You can get more context here https://github.com/shlinkio/shlink-web-client/issues/338

**Reply by luckman212** [reply]
Thank you for that. I wasn't aware of the work being done on the Shlink Dashboard. For now I just bookmarked the base URL of my web client. So it's a couple of extra clicks to get in, but not the end of the world. I mostly interact with my Shlink via a modified version of the CLI based on [this post](https://github.com/shlinkio/shlink/issues/1527#issuecomment-1375540514) anyway.

But I'll be one of the first in line to try the new dashboard when it arrives.
